vmaas/webapp/cache.py

```diff
@@ -149,6 +149,7 @@ def reload(self):
         if self.download():
             self.clear()
             self.load_sqlite(self.filename)
+            self.build_indexes()
```

Comment:
Now there's redundant `self.build_indexes()` in `__init__()` (L:92) .

Reply:
Ah, thanks, fixed.